routes/hosts.js

```diff
@@ -8,17 +8,27 @@ const influx_url = process.env.DOBERVIEW_INFLUX_URI;
 
 
 router.get('/', function(req, res) {
+  let session = req.session;
+  if(session.experiment){
+    db = common.GetMongoDb({exp: session.experiment});
+  } else
+    res.redirect('../');
   var q = url.parse(req.url, true).query;
   var config = common.GetRenderConfig(req);
-  config.hosts = ['apollo', 'calliope', 'clio'];
   config.grafana_sysmon_url = 'http://10.4.73.172:3000/d/WzsbkBwWk/system-mon?orgId=1&kiosk';
-  res.render('hosts', config);
+  db.get("hosts").distinct("name").then(host_list => {
+    config.hosts = host_list;
+    res.render('hosts', config);
+  });
 });
 
+/*
```

Comment:
Comment out or remove this before merging?

Reply:
I don't really know what this does and who put this here. I can uncomment it but it's not used anywhere

Reply:
I actually meant shall we remove it completely if it isn't used. From git blame looks like it was Darryl.

In the end it isn't a biggie either way